src/components/texts/Body-Paragraph.tsx

```diff
@@ -16,6 +16,8 @@ const phrasesState = selector({
 const Sentence = function({ sentence }: { sentence: string }) {
   const phrases = useRecoilValue(phrasesState);
 
+  console.log(phrases);
```

Comment:
It works, I'll just have to delete the older database translations. Did you forget to remove this console.log?

Reply:
I haved removed the console.log.